docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1883 @@
+{
```

Comment:
<p>Perhaps print and mention the <code>x.owner.op</code> to show it is a RandomVariable. We introduced the term in the header but never referred to it afterwards. Also emphasize this is what is exactly the same that is created/returned when we call pm.Normal in the more familiar PyMC model below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
I tried to rephrase it following your suggestion. Feedback is welcome :)